editor/sanitize.js

```diff
@@ -154,17 +153,16 @@ svgedit.sanitize.sanitizeSvg = function(node) {
       var attrName = attr.nodeName;
       var attrLocalName = attr.localName;
       var attrNsURI = attr.namespaceURI;
+
+      // Ignore attributes from custom namespaces
+      var splitNS = attrName.match(/([^:]+):/);
+      if(splitNS && splitNS[1].toUpperCase() in svgedit.ignoredNS) continue;
```

Comment:
Regarding ```toUpperCase()```
I could be wrong, but I believe namespace prefixes in SVG are case sensitive. If I'm right, the svgedit.getReverseNS() function in the current release is also wrong.

That's according to web forums and the only specs I could find. See:
> [SVG 6. Styling](https://www.w3.org/TR/SVG/styling.html#CaseSensitivity) — 6.7 Case sensitivity of property names and values
> [CSS Namespaces Module](https://www.w3.org/TR/2011/REC-css3-namespace-20110929/) — 3.3. Declaring Prefixes